### ISSUE TITLE
Weapon from another mod causes crash when used on Wilden creatures

### ISSUE THREAD

**Issue by Commandore32** [baileyholl/Ars-Nouveau#286]
Version numbers:
Minecraft: 1.16.5
Forge:36.1.23
Immersive engineering: 4.2.4-134
Ars Nouveau version: 1.16.2
Modpack: Direwolf 20 1.16.5 Version: 1.10.1

Crash Report: https://pastebin.com/tVPnkPtQ

I Shot a Wilden Hunter with a Revolver from Immersive using High explosive cartridges. This happens every time. Curiously, when the crash above happened, the game continued to run for 3-5 seconds after the shot was fired, then froze and crashed. This is the first time I've seen this happen out of the 3-4 crashes I've had so far. I was in a sandstone super flat world (my creative test world).

**Comment by baileyholl** [comment]
Does this happen killing other mobs with the revolver, or killing the wilden with other weapons?

**Comment by Commandore32** [comment]
Hi, Bailey. Sorry for the late reply. (All experiments were conducted with high explosive cartridges) I killed mobs from multiple other mods with the revolver, and I killed the Wilden hunter with weapons from multiple other mods, to no effect. The Revolver had no effect on the wilden stalker. One of the immersive engineering devs looked over the crash report and said they found no reference to immersive engineering in the stacktrace. Again sorry for the very late reply.
